src/plugins/tables/reports/pre-a11y-1.json

```diff
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/tables/tables-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Tables",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "wb-tables"
+        }
+    },
+
+    "earl:assertedBy": {
+        "foaf:name": "Service Canada - Principal Publisher",
+        "foaf:homepage": "https://github.com/ServiceCanada",
+        "@type": ["earl:Assertor", "foaf:Organization"],
+        "earl:mainAssertor": {
+            "foaf:name": "Ilya Pak (Github: @ipaksc)",
+            "foaf:homepage": "https://github.com/ipaksc",
+            "@type": ["earl:Assertor", "foaf:Person"]
+        }
+    },
+
+    "dct:date": "2023-06-30",
+
+    "dct:description": "Evaluation of all WCAG 2.1 SC at level AA.",
```

Comment:
Make the same edit as proposed in the other pages

Reply:
done